server/accesslog_test.go

```diff
@@ -121,6 +121,7 @@ func TestHandleHTTPAccessLogs(t *testing.T) {
 			envName:      extAuthzFields[headerEnvironment].GetStringValue(),
 			analyticsMan: testAnalyticsMan,
 		},
+		gatewaySource: "configurable",
```

Comment:
Use a constant for all these instances of string "configurable"?

Reply:
Ack.